view.js

```diff
@@ -0,0 +1,67 @@
+const canvas = document.querySelector("canvas");
+canvas.width = innerWidth;
+canvas.height = innerHeight;
+canvas.style.position = "relative";
+export const c = canvas.getContext("2d");
```

Comment:
odtial ide do "c", ked ho tu premenujes, tak potom ho vies importovat normalnym menom a nemusis davat "c as nieco"

Reply:
nechcelo sa mi prepisovat vsetko v subore preto ten alias ale ak to ma byt prehliadnejsie tak to urobim takto alebo je este jedna moznost ze to exportujem ako default a tam mozem dat nazov aky chcem